src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -344,9 +344,13 @@ const REQUEST_FROM_WARD_STOCKLIST_FIELDS_PUSH_TYPE = {
         }) => ({
           onBlur: () => {
             const valuesWithUpdatedQtyRequested = values;
-            valuesWithUpdatedQtyRequested.lineItems[rowIndex].quantityRequested =
-              values.lineItems[rowIndex].quantityAllowed - fieldValue >= 0 ?
-                values.lineItems[rowIndex].quantityAllowed - fieldValue : 0;
+            const lineItem = valuesWithUpdatedQtyRequested.lineItems[rowIndex];
```

Comment:
Actually, these could be moved to a separate method and passed like the `updateRow`.